starfish/core/image/_filter/test/test_reduce.py

```diff
@@ -18,10 +17,7 @@
 
 
 def make_image_stack():
-    '''
-        Make a test ImageStack
-
-    '''
+    """Make a test ImageStack."""
 
     # Make the test image
     test = np.ones((2, 4, 1, 2, 2), dtype='float32') * 0.1
```

Comment:
could we use imagestack_factory() here instead? Or maybe add a imagestack_from_numpy method to factories/all_purpose.py

Reply:
> could we use imagestack_factory() here instead? 

Plausibly, though the inputs and outputs are very precise and not very numerous.  Not sure if it makes sense to.

> imagestack_from_numpy

ImageStack.from_numpy is basically that, isn't it?